src/GNUmakefile

```diff
@@ -1,66 +1,43 @@
-CFLAGS := -g -W -Wall -Wno-unused-parameter -I. -DGMP -O2 -fno-omit-frame-pointer
-CXXFLAGS := $(CFLAGS) -std=c++11
-
-ifneq (,$(filter $(shell uname -n),elaines-MacBook-Pro.local Nicholass-Air-5.wireless.yale.internal))
-    LIBS := -L/usr/local/lib
-    CFLAGS += -I/usr/local/include
-else
-    LIBS :=
+CFLAGS := -g -W -Wall -Wno-unused-parameter -I. -O2 -fno-omit-frame-pointer
+LIBS := 
+ifndef NGMP
```

Comment:
what's NGMP?

Reply:
It's a flag to compile without GMP. You can run 'make NGMP=1' to compile everything without GMP.